README.md

```diff
@@ -1,7 +1,7 @@
 [![][kong-logo]][kong-url]
 
-[![Build Status][badge-travis-image]][badge-travis-url]
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Kong/kong/blob/master/LICENSE)
+[![Build Status](https://img.shields.io/travis/Kong/kong?logo=Travis%20ci)](https://travis-ci.org/Kong/kong/branches)
```

Comment:
You are missing the `master` branch here. I'm not sure if it would be the default, but while I was checking this the results were different (or at least not in sync).